<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cart</class>
 <widget class="QDialog" name="cart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(236, 229, 199);
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>361</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_next">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(53, 66, 89);
background-color: rgb(236, 229, 199);
</string>
     </property>
     <property name="text">
      <string>صفحه بعدی</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_prev">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(53, 66, 89);
background-color: rgb(236, 229, 199);
</string>
     </property>
     <property name="text">
      <string>صفحه قبلی</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Lalezar&quot;;
color: rgb(53, 66, 89);</string>
     </property>
     <property name="title">
      <string>لیست کالا ها</string>
     </property>
     <widget class="QLabel" name="label_type_8">
      <property name="geometry">
       <rect>
        <x>508</x>
        <y>450</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_g9">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>445</y>
        <width>71</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(116, 141, 166);</string>
      </property>
      <property name="text">
       <string>جزئیات</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_g1">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>44</y>
        <width>71</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(116, 141, 166);</string>
      </property>
      <property name="text">
       <string>جزئیات</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_0">
      <property name="geometry">
       <rect>
        <x>508</x>
        <y>50</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_1">
      <property name="geometry">
       <rect>
        <x>508</x>
        <y>100</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_g2">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>94</y>
        <width>71</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(116, 141, 166);</string>
      </property>
      <property name="text">
       <string>جزئیات</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_g3">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>144</y>
        <width>71</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(116, 141, 166);</string>
      </property>
      <property name="text">
       <string>جزئیات</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_2">
      <property name="geometry">
       <rect>
        <x>508</x>
        <y>150</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_3">
      <property name="geometry">
       <rect>
        <x>508</x>
        <y>200</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_g4">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>194</y>
        <width>71</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(116, 141, 166);</string>
      </property>
      <property name="text">
       <string>جزئیات</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_g5">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>244</y>
        <width>71</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(116, 141, 166);</string>
      </property>
      <property name="text">
       <string>جزئیات</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_4">
      <property name="geometry">
       <rect>
        <x>508</x>
        <y>250</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_5">
      <property name="geometry">
       <rect>
        <x>508</x>
        <y>300</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_g6">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>295</y>
        <width>71</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(116, 141, 166);</string>
      </property>
      <property name="text">
       <string>جزئیات</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_g7">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>345</y>
        <width>71</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(116, 141, 166);</string>
      </property>
      <property name="text">
       <string>جزئیات</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_6">
      <property name="geometry">
       <rect>
        <x>508</x>
        <y>350</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_g8">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>395</y>
        <width>71</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(116, 141, 166);</string>
      </property>
      <property name="text">
       <string>جزئیات</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_7">
      <property name="geometry">
       <rect>
        <x>508</x>
        <y>400</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>59</width>
        <height>35</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>59</width>
        <height>35</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>59</width>
        <height>35</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>59</width>
        <height>35</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>240</y>
        <width>59</width>
        <height>35</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_6">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>290</y>
        <width>59</width>
        <height>35</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>340</y>
        <width>59</width>
        <height>35</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_8">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>390</y>
        <width>59</width>
        <height>35</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_9">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>440</y>
        <width>59</width>
        <height>35</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_0">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_1">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>250</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>300</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>350</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_7">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>400</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_8">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>450</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_10">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>50</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>نام کالا:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_11">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>100</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>نام کالا:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_12">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>150</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>نام کالا:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_13">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>200</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>نام کالا:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_14">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>250</y>
        <width>301</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>نام کالا:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_15">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>300</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>نام کالا:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_16">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>350</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>نام کالا:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_17">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>400</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>نام کالا:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_18">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>450</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>نام کالا:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_10">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>قیمت:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_11">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>قیمت:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_12">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>150</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>قیمت:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_13">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>200</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>قیمت:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_14">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>250</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>قیمت:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_15">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>300</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>قیمت:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_16">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>350</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>قیمت:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_17">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>400</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>قیمت:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_price_18">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>450</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>قیمت:</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_filter">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(53, 66, 89);
background-color: rgb(236, 229, 199);
</string>
     </property>
     <property name="text">
      <string>پرداخت</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton_filter_3">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(53, 66, 89);
background-color: rgb(236, 229, 199);
</string>
     </property>
     <property name="text">
      <string>بازگشت</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
